### ISSUE TITLE
Tool buttons are hard to see in dark mode

### ISSUE THREAD

**Issue by MarianoGnu** [pbaity/rocketchat-dark-mode#209]
**Describe the bug**
tool icons are barely noticeable

**To Reproduce**
1. install dark mode
2. browse to a channel
3. start or open a thread
4. look at to tools

**Expected behavior**
I can distinguish the buttons

**Screenshots**
![chrome_UUE2hpBPc2](https://user-images.githubusercontent.com/1776044/209010777-c3a44bbf-b94c-4149-9927-cb07da473d25.gif)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version 108.0.5359.125 (Oficial Build) (64 bits)

**Additional context**
this was found in a private server that i don't manage

**Comment by MarianoGnu** [comment]
Update, i originally tough the tint came from default theme, but i can see the animation of the icons changing color:
![chrome_keg4IiR967](https://user-images.githubusercontent.com/1776044/209011601-be20f66e-9868-4524-b6ba-2c5d3ad0c84e.gif)

**Comment by raisedadead** [comment]
> this was found in a private server that i don't manage

It would be hard to diagnose if that is the case. 

Here are some possibilities:
- The maintainer of the server should clear their server-side cache, for example, if they are behind a CDN. 
- They need to be on the latest code version from this repo for the corresponding version of Rocket.Chat
- Your browser cache needs updating (you can fix this by clearing the cache or quickly checking the server on a new incognito window).

**Comment by MarianoGnu** [comment]
> * Your browser cache needs updating (you can fix this by clearing the cache or quickly checking the server on a new incognito window).

I tryed this in an incognito window and did not help, also have tryed many devices, and the rocketchat desktop application.
Will contact the server manager with the other suggestions.
If this doesn't work i will probably just close as cannot reproduce

**Comment by pbaity** [comment]
@MarianoGnu is this still a problem?

**Comment by MarianoGnu** [comment]
Yes, i don't know if the system owner did the suggested steps yet tho
